src/App/Controls/ExtendedSlider.cs

```diff
@@ -5,7 +5,7 @@ namespace Bit.App.Controls
     public class ExtendedSlider : Slider
     {
         public static readonly BindableProperty ThumbBorderColorProperty = BindableProperty.Create(
-            nameof(ThumbBorderColor), typeof(Color), typeof(ExtendedSlider), Color.Default);
+            nameof(ThumbBorderColor), typeof(Color), typeof(ExtendedSlider), Color.FromHex("b5b5b5"));
```

Comment:
Define the Color on the styles and then use it here to avoid hardcoding and maybe this color can be reused elsewhere